enterprise/api/v1/views.py

```diff
@@ -231,24 +231,25 @@ def enroll_learners_in_courses(self, request, pk):
         not transactional, in that any one or more failures will not affect other successful enrollments made within
         the same request.
 
-        Expected params:
-            - licenses_info (list of dicts): an array of dictionaries, each containing the necessary information to
-            create a licenced enrollment for a user in a specified course. Each dictionary must contain a user email, a
-            course run key, and a UUID of the license that the learner is using to enroll with.
-            Example:
-                licenses_info: [
-                    {
-                        'email': 'newuser@test.com',
-                        'course_run_key': 'course-v1:edX+DemoX+Demo_Course',
-                        'license_uuid': '5b77bdbade7b4fcb838f8111b68e18ae',
-                    },
-                    ...
-                ]
-
-        Optional params:
-            - discount (int): the percent discount to be applied to all enrollments. Defaults to 100.
-
-        Expected Return Values:
+        Parameters:
+            licenses_info (list of dicts): an array of dictionaries, each containing the necessary information to
```

Comment:
Note: this docstring seems completely wrong: it describes different parameters than the function actually takes.